CHANGELOG.md

```diff
@@ -9,6 +9,13 @@ This project adheres to [Semantic Versioning], with the exception that minor rel
 
 ## [Unreleased]
 
+## [3.2.1] - 2025-08-01
+
+### Fixed
+
+- 🐛 Fix usage of `std::accumulate` ([#1089]) ([**@denialhaag**])
+- 🐛 Fix bug in DD package ([#1088]) ([**@denialhaag**])
```

Comment:
These could be slightly more self explanatory. Someone having problems with the 3.2.0 release coming here to see if the 3.2.1 release fixes things would probably have a harder time than they should.